action_server/src/sema4ai/action_server/_settings.py

```diff
@@ -42,15 +42,15 @@ def get_python_exe_from_env(env):
     return python
 
 
-def _old_get_default_settings_dir() -> Path:
+def _legacy_get_default_settings_dir() -> Path:
     if sys.platform == "win32":
         localappdata = os.environ.get("LOCALAPPDATA")
         if not localappdata:
             raise RuntimeError("Error. LOCALAPPDATA not defined in environment!")
         path = Path(localappdata) / "robocorp" / ".action_server"
     else:
         # Linux/Mac
-        path = Path("~/robocorp/.action_server").expanduser()
+        path = Path("~/.robocorp/.action_server").expanduser()
```

Comment:
@fabioz please verify if we want to update it - I recall talking about missing the `.` was a mistake

Reply:
It is. 

Actually, this might need better handling. 
Not for this PR but there is a good example in Studio how we calculate the path cross-platform. https://github.com/Sema4AI/sema4ai-studio/blob/43e9ba37ce5170b42db59430fa6e92d699fb267a/src/common/pathHandler.ts